### PR TITLE
SectionManagers no longer use container

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by yvesgoeleven** [comment]
@danielmarbach it does not compile on the buildserver though: 

There is no argument given that corresponds to the required formal parameter 'endpointName' of 'EndpointOrientedTopologySectionManager.EndpointOrientedTopologySectionManager(string, NamespaceConfigurations, string, PublishersConfiguration, INamespacePartitioningStrategy, AddressingLogic)'

There is no argument given that corresponds to the required formal parameter 'endpointName' of 'EndpointOrientedTopologySectionManager.EndpointOrientedTopologySectionManager(string, NamespaceConfigurations, string, PublishersConfiguration, INamespacePartitioningStrategy, AddressingLogic)'

**Comment by danielmarbach** [comment]
Hopefully addressed now everything, @Particular/azure-maintainers please review and merge

**Comment by danielmarbach** [comment]
@yvesgoeleven you might want to do another round of review. It is changed drastically since you reviewed it

File: src/Tests/Seam/When_receiving_messages.cs [hunk 1]
```diff
@@ -25,28 +25,29 @@ public class When_receiving_messages
...
             // create the topologySectionManager
             var namespaceManagerCreator = new NamespaceManagerCreator(settings);
 
             var namespaceLifecycleManager = new NamespaceManagerLifeCycleManagerInternal(namespaceManagerCreator);
             var topologyCreator = new TopologyCreator(new AzureServiceBusSubscriptionCreatorV6(topology.Settings.SubscriptionSettings, settings), 
                 new AzureServiceBusQueueCreator(topology.Settings.QueueSettings,settings),  new AzureServiceBusTopicCreator(topology.Settings.TopicSettings), 
                 namespaceLifecycleManager);
-            var topologySectionManager = new EndpointOrientedTopologySectionManager(settings, container);
+
+            var topologySectionManager = container.Resolve<ITopologySectionManagerInternal>();
```
Comment (yvesgoeleven) [inline src/Tests/Seam/When_receiving_messages.cs]: good catch!

**Comment by yvesgoeleven** [comment]
@danielmarbach re-reviewed

**Pull request merged**